Ladybird/ConsoleWidget.cpp

```diff
@@ -38,6 +40,14 @@ ConsoleWidget::ConsoleWidget()
             on_request_messages(0);
     };
 
+    auto* copy_selected_text_action = new QAction("&Copy", this);
+    copy_selected_text_action->setShortcut(QKeySequence(Qt::CTRL | Qt::Key_C));
```

Comment:
Qt isn't my thing, but I know I've seen `StandardKey` used in a few places for standard keyboard shortcuts:

```suggestion
    copy_selected_text_action->setShortcut(QKeySequence::keyBindings(QKeySequence::StandardKey::Copy));
```